src/native/minipal/cpufeatures.c

```diff
@@ -306,6 +306,42 @@ int minipal_getcpufeatures(void)
                 result |= XArchIntrinsicConstants_Serialize;                                               // SERIALIZE
             }
         }
+
+        if (maxCpuId >= 0x07)
+        {
+            // Detect Avx10 here.
+            // Given paradigm:
+            // 1. CPUID.(EAX=07H, ECX=01H):EDX[bit 19] = 1 -> ISA supported.
+            // 2. CPUID.(EAX=24H, ECX=00H):EBX[bits 7:0] >= 1 -> ISA version larger than 1.
+            // with 1,2, set XArchIntrinsicConstants_Avx10_1
+
+            // 3. CPUID.(EAX=24H, ECX=00H):EBX[bits 18:16] -> vector length support.
+            __cpuidex(cpuidInfo, 0x00000007, 0x00000001);
+            if ((cpuidInfo[CPUID_EDX] & (1 << 19)) != 0) // Avx10 supported
+            {
+                __cpuidex(cpuidInfo, 0x00000024, 0x00000000);
+                const int versionMask = 0xFF; // [7:0]
+                if((cpuidInfo[CPUID_EBX] & versionMask) >= 1) // version higher than 1
+                {
+                    result |= XArchIntrinsicConstants_VectorT128; // Avx10 guarantee V128
```

Comment:
This shouldn't be required, `Avx10` requires `Avx2` and so `VectorT128` and `VectorT256` should already be set.

Similarly, since `Avx10/512` implies `Avx512F`, `VectorT512` will also have already been set above.